src/app/docs/summary-action-bar/summary-action-bar-modal-docs.component.html

```diff
@@ -0,0 +1,70 @@
+<sky-modal>
+  <sky-modal-header>
+    Summary action bar in a modal
+  </sky-modal-header>
+  <sky-modal-content>
+    Hello world!
+  </sky-modal-content>
+  <sky-modal-footer>
+    <sky-summary-action-bar>
+      <sky-summary-action-bar-actions>
+        <sky-summary-action-bar-primary-action
+          (actionClick)="instance.save()"
+        >
+          Save
```

Comment:
Is it worth wiring up the primary and secondary buttons to pop up a confirm modal when users select them? I think we used to do that consistently in the past, but without doing a full survey of the component demos, I'm not sure if that's still the case. I found that on the card demo (https://developer.blackbaud.com/skyux-layout/docs/card?docs-active-tab=design), we have a button in the large card that doesn't respond at all, but the experience I'm thinking of is demonstrated in the data entry grid demo (https://developer.blackbaud.com/skyux/components/data-entry-grid). If you select Edit in that demo and then select Save or Cancel, you'll see the confirm modal experience I'm talking about. I'm not sure how valuable that is, but if it is valuable to developers for demonstration purposes and isn't too much work, it might be worth following that pattern here.

Reply:
Good question. That would be placing more emphasis on a "approved" pattern. Is this a pattern we want here @Blackbaud-AdamFunderburk ?

Reply:
Follow-up thought here after looking at other updates... I hadn't thought about testing the responsive behavior in the demo until Adam mentioned it above, but after testing it, I think that makes a stronger case in favor of wiring up some behavior for the secondary buttons. When I decreased the size of the browser so that the secondary actions collapsed into a dropdown, it felt even more awkward for the dropdown menu to remain exposed after clicking the options than the non-response from those buttons felt at full size. Might not change anything, but FWIW, my two cents is that this strengthens the case for wiring up those buttons to at least pop up a confirm modal that indicates they were selected. Curious to hear what Adam thinks though.

Reply:
Talked through this with Alex. He's going to look into the dropdown behavior and either fix the demo or file a bug. As for the secondary actions prompting some sort of visible confirm modal or other behavior, we're going to hold off on that. I'll audit the components demos to see how consistent that behavior is and then revisit the question with UX.